doc/bakugo_spec.md

```diff
@@ -1110,7 +1105,7 @@ if x := f(); x < y {
 ```
 
 ## Return statements
-A "return" statement in a function F terminates the execution of F, and optionally provides one or more result values.
+A "return" statement in a function F terminates the execution of F.
```

Comment:
I noticed definition does not mention returning any values. Could you please explain why this change was made?

Reply:
Oh good catch. I'll revert this